src/components/PostDetails.tsx

```diff
@@ -104,23 +64,17 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && (
+        {loadingComment === true && (
           <Loader />
         )}
 
-        {loaded && hasError && (
-          <div className="notification is-danger" data-cy="CommentsError">
-            Something went wrong
-          </div>
-        )}
-
-        {loaded && !hasError && comments.length === 0 && (
+        {loadingComment === false && !hasError && comments.length === 0 && (
```

Comment:
```suggestion
        {!loadingComment && !hasError && !comments.length && (
```

Reply:
Check all other similar cases.

Reply:
I changed this but it doesn't work I don't know why

Reply:
I mean loadingComment

Reply:
@Vasyliy22 you need to understand why this doesn't work, because the supposed to be working the same